nullaway/src/main/java/com/uber/nullaway/ErrorProneCLIFlagsConfig.java

```diff
@@ -73,6 +78,9 @@
   static final String FL_JI_REGEX_CODE_PATH = EP_FL_NAMESPACE + ":JarInferRegexStripCodeJar";
   static final String FL_ERROR_URL = EP_FL_NAMESPACE + ":ErrorURL";
 
+  static final String AUTO_FIX = EP_FL_NAMESPACE + ":AutoFix";
+  static final String AUTO_FIX_CONFIG_PATH = "/tmp/NullAwayFix/explorer.config";
```

Comment:
We need to get rid of any hardcoded paths. This might take some work, but I don't think we can depend on there being a writable `/tmp/NullAwayFix` directory when running NullAway.